go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/opencontainers/distribution-spec
+
+go 1.14
```

Comment:
should either move up travis to 1.14 or 15 or change this to go 1.13 to match travis test.. See: https://github.com/opencontainers/distribution-spec/blob/master/.travis.yml#L3

Reply:
Sure @mikebrow. Have moved both to 1.15